src/org/labkey/test/tests/SampleSetRemoteAPITest.java

```diff
@@ -181,6 +196,328 @@ public void sampleSetWithMissingValueField() throws IOException, CommandExceptio
+                .addDataField("SampleName", "SampleName", FieldDefinition.ColumnType.String)
+                .addDataField("SampleVolume", "SampleVolume", FieldDefinition.ColumnType.Double)
+                .saveAndClose();
+        insertAssayData(assayName, assaySubfolder, new FieldDefinition.LookupInfo(getProjectName() + "/" + assaySubfolder,
+                "assay.General.AssayForSampleDerivation_assay", "Runs"));
+
+        // get the sampleID, it will be needed later
+        waitAndClickAndWait(Locator.linkWithText("Assay List"));
+        DataRegionTable assayList = DataRegionTable.DataRegion(getDriver()).withName("AssayList").waitFor();
+        CustomizeView customizeView = assayList.openCustomizeGrid();
+        customizeView.showHiddenItems();
+        customizeView.addColumn("ROWID");
+        customizeView.clickSave().save();   // make this the default view
+        String assayIdStringValue = assayList.getRowDataAsMap(0).get("RowId");
+
+        // create sampleset
+        navigateToFolder(getProjectName(), missingValueSamplesFolder);
+        TestDataGenerator dgen = new TestDataGenerator("exp.materials", missingValueTable, getProjectName() + "/" + missingValueSamplesFolder)
+                .withColumnSet(List.of(
+                        TestDataGenerator.simpleFieldDef("name", FieldDefinition.ColumnType.String),
+                        TestDataGenerator.simpleFieldDef("mvStringData", FieldDefinition.ColumnType.String)
+                                .setMvEnabled(true).setLabel("MV Field"),
+                        TestDataGenerator.simpleFieldDef("volume", FieldDefinition.ColumnType.Double)
+                ));
+        dgen.createDomain(createDefaultConnection(true), "SampleSet");
+        dgen.addCustomRow(Map.of("name", "1st", "volume", 13.5));
+        dgen.addCustomRow(Map.of("name", "2nd", "volume", 15.5));
+        dgen.addCustomRow(Map.of("name", "3rd", "volume", 16.5));
+        dgen.addCustomRow(Map.of("name", "4th", "volume", 17.5));
+        dgen.insertRows(createDefaultConnection(true), dgen.getRows());
+
+        // todo: do this using the SaveAssayBatchCommand; it currently blows up if you attempt to get a Material without supplying the SampleSetID
+        String createScript = "LABKEY.Experiment.saveBatch({\n" +
+                "  assayId: "+assayIdStringValue+",\n" +
+                "  batch: {\n" +
+                "    name: 'foo',\n" +
+                "    runs: [{\n" +
+                "      name: 'bar',\n" +
+                "\n" +
+                "      materialOutputs: [\n" +
+                "        // create some new samples in the target sample set if it doesn't exist\n" +
+                "        { name: 'a new one', sampleSet: { name: 'mvAssaySamples' }, properties: { 'mvStringData': 'new from api' } },\n" +
+                "        { name: 'another new one', sampleSet: { name: 'mvAssaySamples' }, properties: { 'mvStringData': 'also new from api' } }\n" +
```

Comment:
For a missing value test wouldn't it be useful to set the mvStringData, and mvstringdata_mvindicator appropriately?

Reply:
yes, it would.  And it fails; so new bug.